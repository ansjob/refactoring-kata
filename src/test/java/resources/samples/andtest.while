
# I'm a comment!

if true & false then
 x:=0
else
 x:=1