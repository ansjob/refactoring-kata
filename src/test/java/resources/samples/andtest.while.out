=== Running from {x=NEG}
<[L0] FALSE:[L0] TRUE:[L0] AND:[L0] BRANCH([[L1] PUSH-0, [L1] STORE-x], [[L2] PUSH-1, [L2] STORE-x]):ε, ε, s[x→NEG]>
<[L0] TRUE:[L0] AND:[L0] BRANCH([[L1] PUSH-0, [L1] STORE-x], [[L2] PUSH-1, [L2] STORE-x]):ε, FF:ε, s[x→NEG]>
<[L0] AND:[L0] BRANCH([[L1] PUSH-0, [L1] STORE-x], [[L2] PUSH-1, [L2] STORE-x]):ε, TT:FF:ε, s[x→NEG]>
